sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/Fixtures/IaasLinuxPoolFixture.cs

```diff
@@ -30,6 +31,20 @@ public static ImageInformation GetUbuntuImageDetails(BatchClient client)
             return ubuntuImage;
         }
 
+        public static ImageInformation GetUbuntuServerImageDetails(BatchClient client)
+        {
+            List<ImageInformation> imageInformation = client.PoolOperations.ListSupportedImages().ToList();
+
+            static bool ubuntuImageScanner(ImageInformation imageInfo) =>
+                imageInfo.ImageReference.Publisher.ToLower().Contains("canonical") &&
+               imageInfo.ImageReference.Offer.Contains("ubuntu");// &&
+               // imageInfo.ImageReference.Sku.Contains("18.04");
```

Comment:
Seems like we should be filtering on version somehow here, even if it's just "latest"

Reply:
fixed